x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/csp_boxed_radio_group.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useEuiTheme, EuiButton, EuiRadio } from '@elastic/eui';
+import { css } from '@emotion/react';
+
+interface Props {
+  disabled?: boolean;
+  options: RadioOption[];
+  onChange(id: string): void;
+  idSelected: string;
+  size?: 's' | 'm';
+}
+
+interface RadioOption {
+  disabled?: boolean;
+  id: string;
+  label: string;
+  icon?: string;
+  tooltip?: string;
+}
+
+export const RadioGroup = ({ idSelected, size, options, disabled, onChange }: Props) => {
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <div
+      css={css`
+        display: flex;
+        flex-flow: row wrap;
+        gap: ${euiTheme.size.s};
+
+        // Show rows for m- screens (below 768px)
+        @media only screen and (max-width: ${euiTheme.breakpoint.m}px) {
+          button {
+            min-width: 100%;
+          }
+        }
+      `}
+    >
+      {options.map((option) => {
+        const isChecked = option.id === idSelected;
+        return (
+          <EuiButton
+            disabled={option.disabled || disabled}
+            style={{
+              flex: 1,
+              border: `1px solid ${
```

Comment:
you can use `euiTheme.border.thin` instead of `1px solid` if you want

Reply:
<img width="369" alt="Screen Shot 2023-01-02 at 11 20 06" src="https://user-images.githubusercontent.com/20814186/210212963-a218a4df-28a0-4fa0-93a9-b8bae369b253.png">

it's only good for 1 color, we need both.